.github/workflows/run-sugar-cli-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Run Sugar CLI Test
+
+on:
+  workflow_dispatch:
+
+env:
+  SOLANA_VERSION: 1.9.22
+  RUST_TOOLCHAIN: stable
+
+jobs:
+  run-sugar-cli-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/install-linux-build-deps
+      - uses: ./.github/actions/install-solana
+        with:
+          solana_version: ${{ env.SOLANA_VERSION }}
+      - uses: ./.github/actions/install-rust
+        with:
+          toolchain: ${{ env.RUST_TOOLCHAIN }}
+
+      - run: solana-keygen new -o $GITHUB_WORKSPACE/kp --no-bip39-passphrase
+        shell: bash
+
+      - run: solana config set --url https://api.devnet.solana.com --keypair $GITHUB_WORKSPACE/kp
```

Comment:
should i swap this with a non-public endpoint?

Reply:
ok cool, that's what i was thinking too - will swap out for a genesys go endpoint (https://devnet.genesysgo.net)

Reply:
good point on the local validator - i'll ask someone more familiar with sugar if that's possible. but, at first glance, it looks like we only expect to run against devnet or mainnet https://github.com/metaplex-foundation/sugar/blob/main/script/sugar-cli-test.sh#L181-L194

if not, another option for to mitigate false negatives due to network conditions is this [retry action](https://github.com/marketplace/actions/retry-step).